Rakefile

```diff
@@ -13,21 +13,3 @@ RDoc::Task.new(:rdoc) do |rdoc|
   rdoc.rdoc_files.include('README.md')
   rdoc.rdoc_files.include('lib/**/*.rb')
 end
-
-APP_RAKEFILE = File.expand_path('../test/migration/Rakefile', __FILE__)
-load 'rails/tasks/engine.rake'
-
-load 'rails/tasks/statistics.rake'
-
-require 'bundler/gem_tasks'
-
-require 'rake/testtask'
-
-Rake::TestTask.new(:test) do |t|
```

Comment:
do we really want to remove the testing framework for rails? (not sure if we wont end some ruby code at some point?)

Reply:
I would prefer to add it if we need it than keep it just in case.

Reply:
And it doesn't work as is.
```
rake aborted!
LoadError: cannot load such file -- miq_v2v_ui_plugin
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/test/migration/config/application.rb:6:in `require'
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/test/migration/config/application.rb:6:in `<top (required)>'
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/test/migration/Rakefile:4:in `require_relative'
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/test/migration/Rakefile:4:in `<top (required)>'
/usr/local/lib/ruby/gems/2.3.0/gems/railties-5.0.6/lib/rails/tasks/engine.rake:3:in `load'
/usr/local/lib/ruby/gems/2.3.0/gems/railties-5.0.6/lib/rails/tasks/engine.rake:3:in `block (2 levels) in <top (required)>'
/usr/local/lib/ruby/gems/2.3.0/gems/railties-5.0.6/lib/rails/tasks/engine.rake:2:in `block in <top (required)>'
/usr/local/lib/ruby/gems/2.3.0/gems/railties-5.0.6/lib/rails/tasks/engine.rake:72:in `<top (required)>'
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/Rakefile:18:in `load'
/Users/zita/Desktop/ManageIQ/miq_v2v_ui_plugin/Rakefile:18:in `<top (required)>'
/usr/local/lib/ruby/gems/2.3.0/gems/rake-11.3.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => load_app
(See full trace by running task with --trace)
```